source/collections/capped-collection.txt

```diff
@@ -0,0 +1,55 @@
+.. _capped-collection:
+
+==========================
+Create a Capped Collection
+==========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+:manual:`Capped Collections </core/capped-collections/>` are fixed-sized 
+collections that support high-throughput operations that insert and retrieve 
+documents based on insertion order. 
+
+Procedure
+---------
+
+.. procedure::
+   :style: connected
+
+   .. step:: Click the :guilabel:`Create Collection` button.
+
+      From the :guilabel:`Collections` screen, click the 
+      :guilabel:`Create Collection` button.
+
+   .. step:: Enter the collection name.
+
+   .. step:: Click the :guilabel:`Advanced Collection Options` dropdown.
+
+      Check the :guilabel:`Capped Collection` option.
+
+   .. step:: Enter the :guilabel:`size` of the capped collection.
+
+      Enter the maximum number of bytes that the collection can hold.
+
+   .. step:: Click :guilabel:`Create Collection` to create the collection.
+
+Restrictions and Limitations
+----------------------------
+
+The following restrictions and limitations apply when creating a 
+capped collection:
+
+- You can only use :guilabel:`custom collation` alongside your capped 
+  collection from the :guilabel:`Advanced Collection Options`. 
```

Comment:
I think I know what this is trying to say but it sounds ambiguous. At first read, I thought this meant that “custom collation”, having been an option elsewhere as well, is now _only_ available in the “Advanced Collection Options” section. But this is saying that “custom collation” is, out of all advanced options, the _only_ one that is still available, right? Can we reword this?

Reply:
Done.